### DISCUSSION TITLE
Bug:The issue with calling Zhipu AI is that in the chat assistant type, workflow orchestration is available, but errors occur when using the workflow type.

### DISCUSSION THREAD

**Discussion by ixujue** [langgenius/dify#3583] [Help]
The issue with the ZhiPu AI model arises when operating within the chat assistant type where workflow orchestration is functioning properly. However, errors are encountered when switching to the workflow type. Among the four available models, only one model operates correctly in the workflow type (chatglm_turbo). The other two models result in errors (glm-3-turbo, glm-4), and one model seems to be in a state of self-entertainment (glm-4v), consistently producing hallucinatory results. There are two specific error prompts that occur: "Node LLM run failed: [Zhipu] Error: Error code: 400, with error text {"error":{"code":"1214","message":"The messages parameter is illegal. Please check the document."}}. When this error appears, the functioning model remains unaffected, but selecting either of the other two models leads to the error.

![不可用模型的报错结果](https://github.com/langgenius/dify/assets/48663394/4032efb6-cb80-41a1-bfd6-ea9cd611baef)
use glm-3-turbo, glm-4

![不可用模型的幻觉结果](https://github.com/langgenius/dify/assets/48663394/a344e6a9-af77-46eb-b02b-e517e97041c3)
use glm-4v

![可用模型](https://github.com/langgenius/dify/assets/48663394/fade09fc-aeaa-41b5-9fa6-9ff37eafa300)
use chatglm_turbo

![可用模型的正常结果01](https://github.com/langgenius/dify/assets/48663394/373f9b84-f1a3-4f80-a6e3-bb9be9f43ff8)

![可用模型的正常结果02](https://github.com/langgenius/dify/assets/48663394/436fbe6d-b322-4d34-b0b7-0c582b814e01)

**Comment by crazywoola** [comment]
Please go to issues to fire a bug report.
